argusclient/client.py

```diff
@@ -591,6 +591,72 @@ def get_alerts_allinfo(self, ownerName=None, alertNameContains=None, shared=Fals
         """
         return self.argus._request("get", "alerts/allinfo", params=dict(ownername=ownerName, alertNameContains=alertNameContains, shared=shared, limit=limit))
 
+    '''
+    Functions for enabling composite alerts
+    '''
+
+    def get_composite_alert_children_info(self, alert_id):
+        if not alert_id: raise ValueError("Need to specify alert id")
+        uri_path = "alerts/{}/children/info".format(alert_id)
+        return self.argus._request("get", uri_path)
+
+    def create_composite_alert(self, payload):
+        if not payload: raise ValueError("Need to specify an payload")
+        alert_obj = self.argus._request("post", "alerts", dataObj=payload)
+        self._coll[alert_obj.id] = alert_obj
```

Comment:
Is this a complete alert object with all the fields?

Reply:
Could you please clarify this?

Reply:
Removed this function, as we are using add(alert_obj)